### DISCUSSION TITLE
Ignore Empty Buffer

### DISCUSSION THREAD

**Discussion by semi710** [olimorris/persisted.nvim#83] [Q&A] [answered]
How to ignore the buffer from storing in session if the buffer is empty with "[No Name]" ?
happens to me if i close all buffers and then nvim goes to a empty buffer that get's saved to session

**Comment by olimorris** [comment, marked as answer]
Use the `should_autosave` callback

**Reply by semi710** [reply]
My callback fn is
```lua
function()
-- do not autosave if the alpha dashboard is the current filetype
    if vim.bo.filetype == "alpha" then
        return false
    end
    return true
            end,
```

how can i check for a blank buffer?

**Reply by olimorris** [reply]
Check for no filetype?

**Reply by semi710** [reply]
if i do `vim.bo.filetype` on a empty buffer it returns `""` but if i do it on something that is not having an extension 
e.g sometimes in a hurry i type a file name `text` only the file type of this file is also `""`
